addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -24,7 +24,8 @@
                 </button>
             </xpath>
             <xpath expr="//header" position="inside">
-                <button name="action_make_billable" string="Create Sales Order" type="object" attrs="{'invisible': [('display_create_order', '=', False)]}" groups="sales_team.group_sale_salesman"/>
+                <!-- To be removed in master -->
+                <button name="action_make_billable" string="Create Sales Order" type="object" attrs="{'invisible': [('display_create_order', '=', False)]}" groups="sales_team.group_sale_salesman" class="d-none"/>
```

Comment:
You can also do this:
```suggestion
                <button name="action_make_billable" string="Create Sales Order" type="object" groups="sales_team.group_sale_salesman" invisible="1"/>
```
By the way, I prefer the `class` attribute is the first one in a html tag or after an `id` attribute (not the case for you) 😇